zh/editor/assets/index.md

```diff
@@ -20,14 +20,14 @@
 
     - 复制：Ctrl or Cmd + C
     - 粘贴：Ctrl or Cmd + V
-    - 拷贝：Ctrl or Cmd + D，Ctrl + 拖动资源
+    - 克隆：Ctrl or Cmd + D，Ctrl + 拖动资源
     - 删除：Delete
     - 上下选择：上下箭头
     - 文件夹的折叠：左箭头
     - 文件夹的展开：右箭头 
     - 多选：Ctrl or Cmd + 点击
     - 多选：Shift + 点击
-    - 重命名： Enter
+    - 重命名： Enter / F2
```

Comment:
Enter / F2
-->
Enter/F2

Reply:
The above changes have updated. 
And add in En docs:
```
    - **Rename**: Enter/F2
    - **Cancel Rename**: Esc
```